pokemon-checker/src/services/AbilityService.test.ts

```diff
@@ -31,7 +31,9 @@ test("Test effect fetched properly", () => {
 });
 
 test("Test pokemonList fetched properly", () => {
-  expect(fetchedAbility.pokemons).toEqual(pressureAbility.pokemons);
+  expect(fetchedAbility.id).toEqual(pressureAbility.id);
```

Comment:
Noticed the number of pokemon that hold "pressure" went from 32 to 36 since this test was created.

I've moved the check to compare against more static things like `id`, `name`, and `url`.